File: exercises/ex5/demo/spatial.c
#include <stdio.h>

int main() {
    int index;
    int array[] = {1, 2, 3, 4, 5, 6};

    printf("input array index: ");
    scanf("%d", &index);
    printf("array[%d] = %d\n", index, array[index]);

    return 0;
}

File: exercises/ex5/demo/rust/src/spatial.rs
use scanf::scanf;

fn main() {
    let mut index = 0;
    let array = [1, 2, 3, 4, 5, 6];

    print!("input array index: ");
    scanf!("{}", index).unwrap();

    println!("array[{}] = {}", index, array[index]);
}
